docs/src/pages/lab/slider/slider.md

```diff
@@ -37,7 +37,10 @@ Sliders reflect the current state of the settings they control.
 
 ## Reverse slider
```

Comment:
Why not remove this section all together?

Reply:
I was thinking about people looking for the missing prop but the changelog entry and this PR is probably sufficient.